<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoFrameClass</class>
 <widget class="QWidget" name="PhotoFrameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotoFrame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="photoWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#photoWidget { 
border-width: 5px; 
border-style: outset; 
border-color: #FFFF00; 
border-radius: 10px; 
/*background: #FFFFFF; */
color: #000000; 
selection-background-color:#000000; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="defaultBorder">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#defaultBorder { 
border-width: 5px; 
border-style: outset; 
border-color: #DDDDDD; 
border-radius: 10px; 
/*background: #FFFFFF; */
color: #000000; 
selection-background-color:#000000; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="selectedBorder">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>50</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#selectedBorder { 
border-width: 5px; 
border-style: outset; 
border-color: #FFFF00; 
border-radius: 10px; 
/*background: #FFFFFF; */
color: #000000; 
selection-background-color:#000000; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
